### DISCUSSION TITLE
Pouvoir du traitre

### DISCUSSION THREAD

**Discussion by ElethornGM** [fibreville/minotaure_php#80] [Boite à idées]
Salut, une idée proposée par plusieurs viewers ce soir : que si le traitre utilises son pouvoir (qui permet de supprimer l'un des choix possibles dans un sondage), tous les joueurs ayant déjà voté pour cette option récupère leur vote, et peuvent voter pour un nouveau choix !
ça permettra d'éviter les petits malins qui attendent la fin d'un sondage et suppriment le choix qui a été choisi à 80%, ce qui est assez frustrant pour tout le monde...

**Comment by Seikame** [comment]
C'est pas bête ça, après ça change beaucoup l'approche et l'impact que le rôle a. Le must serait de pouvoir choisir entre l'un ou l'autre, mais quitte à choisir si c'est trop compliqué de laisser les deux j'aurai aussi une très légère préférence pour la redistribution des votes.

**Reply by ElethornGM** [reply]
Ouai, parce que là vraiment pendant mon live, 3 ou 4 fois les joueurs avec le pouvoir du traitre ont attendu que 80% des gens aient voté, et c'est vraiment fourbe comme technique ^^

**Reply by uurshin** [reply]
En fait c'était moins le cas sur la V1 car le MJ pouvait choisir de ne pas recharger et ainsi cacher les votes pendant un moment. Ça me parait être une bonne idée. On peut effectivement le rendre paramétrable mais je sais pas si ça a du sens : si on estime qu'une fonctionnalité est foireuse c'est peut être mieux de proposer uniquement la "meilleure" version, même si c'est très subjectif. J'aimerai avoir l'avis de Happy1000 sur le sujet aussi.

Et à part ça, je pense que ce serait intéressant de découpler vote et utilisation du pouvoir aussi (pour permettre de voter et d'utiliser le pouvoir sur un autre choix).

**Reply by HappyFR** [reply]
De mon coté je trouve que c'est tout l'intérêt de la feature que les votes soit "gachés" pour ceux qui ont voté pour le choix supprimé. Si les joueurs peuvent revoter derrière avec un choix en moins je trouve que le traitre à beaucoup moins d'impact, et ça créer moins de surprise ou de situations amusante. Et en terme de rythme de jeu, je trouve ça un peu redondant. 

Par contre, effectivement le choix du traitre est drivé par le vote en cours et c'est un réel problème. 
Comme dit Uurshin, peut-etre qu'en camouflant le vote on réglerai le problème. Laisser le pourcentage de votant en visible, avec un bouton "découvrir les votes" pourrait marcher. 
Après le voleur pourrait encore attendre que le MJ clique sur le bouton avant d'utiliser son pouvoir (vu que ce bouton ne terminerai pas le sondage mais le révèlerai juste) donc il faudrait ajouter une règle qui fait que son pouvoir ne peut-être utilisé que lorsque les résultats sont cachés.

**Reply by uurshin** [reply]
Intéressant ! Perso j'aime bien le fait de voir la progression des votes au fur et à mesure, ça fait plus "discussion du groupe avec les avis qui sont émis les uns après les autres", mais dévoiler les votes avec un clic me semble plus proche d'un vrai sondage pour garder le suspens.
On pourrait proposer "sondage ouvert" et "sondage caché" comme boutons, et lors des sondages ouverts le traitre ne pourrait pas intervenir ?

**Reply by ElethornGM** [reply]
Yes ce serait une bonne solution je trouve uurshin

**Reply by HappyFR** [reply]
J'aime beaucoup!

**Comment by asaurat** [comment]
Ce serait cool mais au final je pense que pour les prochaines sessions je cacherai l'écran sondage pendant le stream. C'est dommage de montrer ces infos, traître ou pas (quand on est parmi les derniers à voter on est forcément influencé par les votes précédents si on les voit).

Le fait de cacher ne règle pas le souci du traître "trop" frustrant, mais en tant qu'animateur j'essaierai de pousser le traître à agir vite =)

Reste que ce serait sympa à coder (mais perso je vais plancher sur d'autres sujets d'abord).
